firefox-ios/Client/TabManagement/TabManagerImplementation.swift

```diff
@@ -189,16 +189,16 @@ class TabManagerImplementation: LegacyTabManager, Notifiable, WindowSimpleTabsPr
                    level: .debug,
                    category: .tabs)
 
-        selectTab(tabToSelect)
```

Comment:
There was an edge case where tabToSelect might be nil so I thought we should handle that explicitly.